### DISCUSSION TITLE
budibase helm hijack cluster wide cert-manager

### DISCUSSION THREAD

**Discussion by jalberto** [Budibase/budibase#4731] [Help]
I installed budibase by helm in its own namespace for testing, after I was done I removed budibase and its namespace, but now my cert manager insist in using 'budibase-ingress-nginx-controller-admission` for every ingress.httproxy in the cluster, and it's causing a lot of problems.

Where is this setup? how can I recover previous configuration?

**Comment by jcsrb** [comment]
we found that budibase created a `ValidatingWebhookConfiguration` that was not removed 

```
❯ kubectl get ValidatingWebhookConfiguration
NAME                                         WEBHOOKS   AGE
budibase-ingress-nginx-admission             1          34d
cert-manager-webhook                         1          58m
validation-webhook.snapshot.storage.k8s.io   1          172d
```

it appeared to have higher order/precedence  then the cert-manager webhook
removing it solved the issue

```
❯ kubectl delete ValidatingWebhookConfiguration budibase-ingress-nginx-admission
```

**Comment by shogunpurple** [comment]
Ah - I think I see what is happening here. There was a bug in the nginx-ingress-controller, that I ran into also on older kubernetes versions. The configuration for NGINX ingress isn’t handled by us, as it’s part of the official NGINX ingress controller chart.

What kubernetes version are you running in your cluster?

**Reply by jalberto** [reply]
1.22.7

also the whole installation was made in its own namespace, so it was a bigger surprise to find something out of it
